Packs/RubrikPolaris/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,70 @@
+
+#### Integrations
+##### Rubrik Radar
+- Modified the graphql query during incident ingestion to filter on the 'lastUpdated' time for new incidents in accordance to backend functionality changes to Radar.  This also fixed a bug where new incidents were not being detected by XSOAR.
+- Added two new options in the Rubrik Polaris integration settings that allow the user to map the Radar Critical and Warning event severity levels to various XSOAR severity levels upon incident fetch.  The default severity mappings are currently:  Radar Critical -> XSOAR HIGH and Radar Warning -> XSOAR LOW.
+- Added the 'rubrik-cdm-cluster-connection-state' and 'rubrik-cdm-cluster-location' commands.  These commands have been added in as tasks to the Rubrik Post Intrusion Ransomware Investigation playbook.
+- Added the Rubrik CDM Cluster Location and Rubrik CDM Cluster Name custom fields.
+- Added functionality to pull in the Radar File Change attributes from Radar events during incident ingestion.
+- Various layout changes were made to the Rubrik Radar Incident Summary page.
```

Comment:
this is mentioned later - we can remove this line.

Reply:
These lines have been removed.